akka-projection-testkit/src/main/scala/akka/projection/testkit/javadsl/ProjectionTestKit.scala

```diff
@@ -42,9 +44,9 @@ final class ProjectionTestKit private[akka] (testKit: ActorTestKit) {
    * If the assert function doesn't complete without error within 3 seconds the test will fail.
    *
    * @param projection - the Projection to run
-   * @param assertFunction - a Runnable that exercise the test assertions
+   * @param assertFunction - a function that exercise the test assertions
    */
-  def run(projection: Projection[_], assertFunction: Runnable): Unit =
+  def run[R](projection: Projection[_], assertFunction: Creator[R]): R =
```

Comment:
`R` makes it more difficult for user to write the lambda, so are we sure that it's useful to return value?

Reply:
See my comment [here](https://github.com/akka/akka-projection/pull/162#issuecomment-632752384). I will review or justify it.